src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -248,7 +248,7 @@ private void CreateRequestBuilderClass(CodeNamespace currentNamespace, OpenApiUr
         foreach (var child in currentNode.Children)
         {
             var propIdentifier = child.Value.GetClassName();
-            var propType = propIdentifier + requestBuilderSuffix;
+            var propType = child.Value.DoesNodeBelongToItemSubnamespace() ? propIdentifier + "Item"+ requestBuilderSuffix : propIdentifier + requestBuilderSuffix;
```

Comment:
In the beta openApi, some request builders e.g. `ActivityStatisticsItemRequestBuilder` resulted into a requestBuilder with the same name in the `Item` namespace. This caused issues with ambiguity using that description as the class initialized a collection indexer for the item requestBuilder.